manifest_staging/charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -126,6 +126,11 @@ spec:
       {{- if .Values.audit.priorityClassName }}
       priorityClassName:  {{ .Values.audit.priorityClassName }}
       {{- end }}
+      securityContext:
+        {{- toYaml .Values.audit.podSecurityContext | nindent 8 }}
+        fsGroup: 999
+        supplementalGroups:
+        - 999
```

Comment:
This should be removed since we are already injecting the security context through the helm value above. You can remove these fields in `cmd/build/helmify/kustomization.yaml`